### ISSUE TITLE
More stringent declarations are needed

### ISSUE THREAD

**Issue by JHeLiu** [dotnet/AspNetCore.Docs.zh-cn#292]
[在此处输入反馈]
When we come from the.NET Framework developers, most of the web applications are developed using synchronization, while under.NET CORE, it is more asynchronous. We hope to have a clear description, such as whether the benefits of asynchrony are greater than that of synchronization, or asynchrony is necessary, otherwise developers will be easily confused.For example, in highly concurrent WEB applications, asynchrony and synchronization should be preferred

---
#### 文档详细信息

⚠ *请勿编辑此部分。 docs.microsoft.com ➟ GitHub 问题链接必须具有此部分。*

* ID: 9599d917-4e43-f1ae-452a-1f11aaa985ed
* Version Independent ID: 5cc05b6c-c671-3607-3217-7db73ebc1f28
* Content: [ASP.NET Core 性能最佳做法](https://docs.microsoft.com/zh-cn/aspnet/core/performance/performance-best-practices?view=aspnetcore-3.1)
* Content Source: [aspnetcore/performance/performance-best-practices.md](https://github.com/dotnet/AspNetCore.Docs.zh-cn/blob/live/aspnetcore/performance/performance-best-practices.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-performance**
* GitHub Login: @mjrousos
* Microsoft Alias: **riande**

**Comment by mjrousos** [comment]
In the 'Avoid blocking calls' section of the doc, we do say that hot code paths should be asynchronous since async behavior is preferred for highly concurrent scenarios like web services. Do you think that message needs to be clearer? Or is it maybe not translated well?

**Comment by JHeLiu** [comment]
@mjrousos  
Yes, it's not that the translation is bad, because there is a special highlight tag, such as how to use ASP NET CORE under high concurrency,
Such as

**How is high concurrency developed**
